src/components/button/Button.types.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement, FunctionComponent } from 'react'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Member 'FunctionComponent' of the import declaration should be sorted alphabetically.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)